public/scripts/location.js

```diff
@@ -122,6 +123,7 @@ const map = new ol.Map({
     center: ol.proj.fromLonLat([mapInitialLon, mapInitialLat]),
     zoom: !isDetailsScreen && !isMapResultsScreen ? extentSearchZoomLevel : 2,
     minZoom: 2,
+    extent: ol.proj.get('EPSG:3857').getExtent(),
```

Comment:
what is this value refers to ? can you put this in a meaningful variable and use it here?